doc/classes/SplitContainer.xml

```diff
@@ -33,6 +59,16 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="drag_released">
+			<description>
+				Emitted when the mouse is released from the drag area.
+			</description>
+		</signal>
+		<signal name="drag_started">
+			<description>
+				Emitted when the mouse is pressed inside the drag area.
```

Comment:
```suggestion
				Emitted when the user started dragging.
```

It's more about the drag starting than the mouse being pressed.

Reply:
See above comment regarding the signal names - I will edit for clarity.